pkg/testcase/pod.go

```diff
@@ -59,3 +59,13 @@ func processPodStatus(
 		}
 	}
 }
+
+// TestPodStatusIsRunningByLabel checks status of Running pods when searched by label 
+func TestPodStatusIsRunningByNamespaceAndLabel(namespace, label string) {
```

Comment:
this is a generic func, should not be linving in the same file as the test.

please lets use shorter and concise names as golang patterns.

Reply:
Updated